packages/hint-create-element-svg/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Infrom users that they need to use createElementNS to create SVG elements instead of createElement",
+  "devDependencies": {
+    "@hint/parser-javascript": "^2.0.3",
+    "@hint/utils-tests-helpers": "^2.0.3",
+    "@types/debug": "0.0.31",
+    "@types/node": "10.12.18",
+    "ava": "^0.25.0",
```

Comment:
This version is wrong and should be 1.x something. This is probably related to the issue you were having running `profile.js`. There will probably be some other outdated dependencies. Can you please update them?
I'll try to do a release today with the updated `create-hint` package if #1937, #1944 and #1950 get merged soon.
@antross can you please take a look at those when you have time?

Reply:
@molant This fixed the ava issue I was having -- thanks! :)